mixer/pkg/runtime/dispatcher/session.go

```diff
@@ -142,8 +143,18 @@ func (s *session) dispatch() error {
 			ndestinations++
 
 			for j, input := range group.Builders {
+				if s.variety == tpb.TEMPLATE_VARIETY_QUOTA {
+					// only dispatch instances with a matching name
```

Comment:
This looks good, and the comment here is enough. But really should call out this behavior somewhere. Does the op-facing documentation needs to be updated?